joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

left_back_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: left_back_leg_joint_1

left_back_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: left_back_leg_joint_2

left_mid_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: left_mid_leg_joint_1

left_mid_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: left_mid_leg_joint_2

left_front_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: left_front_leg_joint_1

left_front_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: left_front_leg_joint_2

right_back_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: right_back_leg_joint_1

right_back_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: right_back_leg_joint_2

right_mid_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: right_mid_leg_joint_1

right_mid_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: right_mid_leg_joint_2

right_front_leg_1_controller:
  type: "position_controllers/JointPositionController"
  joint: right_front_leg_joint_1

right_front_leg_2_controller:
  type: "position_controllers/JointPositionController"
  joint: right_front_leg_joint_2